src/partials/_typography.scss

```diff
@@ -0,0 +1,10 @@
+@font-face {
+  font-family: "Pacifico";
+  src: url(./fonts/Pacifico/Pacifico-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: url(./fonts/Montserrat/Montserrat-Italic-VariableFont_wght.ttf), url(./fonts/Montserrat/Montserrat-VariableFont_wght.ttf);
```

Comment:
Great looking site!! Maybe use gitignore for the fonts that are not being used

Reply:
awesome feedback! thanks James! 😁